Vector/ViewController/RoomViewController.m

```diff
@@ -109,6 +116,32 @@ + (instancetype)roomViewController
 
 #pragma mark -
 
+- (instancetype)initWithNibName:(nullable NSString *)nibNameOrNil bundle:(nullable NSBundle *)nibBundleOrNil
```

Comment:
It may worth to make a generic method (the same as applyDefaultConfig MXKRoomViewController.m)  at MXKViewController level to setup default values.